docs/UserGuide.md

```diff
@@ -90,73 +91,79 @@ interactions recorded faster and simpler than traditional GUI applications.
+* `add n/John a/Yishun Street 81 p/9876543 e/john@yahoo.com t/friends t/supplier`
 
 ### Clearing all entries : `clear`
 
 Clears all entries from JeeqTracker.
 
 Format: `clear`
 
-* A confirmation message will appear after the command.
-* Input `confirm` to proceed with the clearing of all entries.
+* All entries that are cleared cannot be retrieved.
 
 Example:
 
-* `clear` followed by `confirm`
+* `clear` Clears all Client's entries together with the Remarks and Transactions.
 
-### Creating a Remark: `create`
+### Creating a Remark: `remark`
 
-Creates a new remark (remark) for a specific client.
+Creates a new remark for the specified client.
 
-Format: `create INDEX n/NAME p/NUMBER e/EMAIL [t/TAG]`
+Format: `remark INDEX REMARK`
 
-* Tags are optional.
-* Multiple tags can be tagged to the remark.
-* The index refers to the index number shown in the displayed client list.
-* The index **must be a positive integer** 1, 2, 3, …​
+* The `INDEX` refers to the index number shown in the displayed client list.
+* The `INDEX` **must be a positive integer** 1, 2, 3, …​
+* The parameter `REMARK` cannot be empty.
 
 Examples:
-* `create 1 n/James p/82692192 e/JamesSho@KFC.com` adds James as a remark for the client at index 1.
-* `create 5 n/John p/95692192 e/John123@ColdStorage.com t/Manager t/MainRemark` tags can also be added.
+* `remark 1 punctual buyer` adds the remark `punctual buyer` to the client at index 1.
+* `remark 5 fast and decisive` adds the remark `fast and decisive` to the client at index 5.
+
+### Deleting a client / transaction / remark : `delete`
 
-### Deleting a client : `delete`
+Deletes the specified `client`, `transaction` or `remark` from JeeqTracker.
 
-Deletes the specified client from JeeqTracker.
+Format: `delete INDEX m/MODE`
 
-Format: `delete INDEX`
+* The `INDEX` refers to the index number shown in the displayed list.
+* The `INDEX` **must be a positive integer** 1, 2, 3, …​
+* The `MODE` refers to which entity is being deleted. It must be `client`, `transaction` or `remark`.
+* `view` command must be used before the deletion of `remark` or `transaction` as the deletion can only happen when they are visible in the application. 
 
-* Deletes the client at the specified `INDEX`.
-* The index refers to the index number shown in the displayed client list.
-* The index **must be a positive integer** 1, 2, 3, …​
 
 Examples:
-* `list` followed by `delete 2` deletes the 2nd client in the address book.
-* `find cold` followed by `delete 1` deletes the 1st client in the results of the `find` command.
+* `list` followed by `delete 2 m/client` deletes the 2nd client in the JeeqTracker.
+* `find cold` followed by `delete 1 m/client` deletes the 1st client in the results of the `find` command.
+* `list` followed by `view 2` displays the remarks and transactions of the 2nd client in the JeeqTracker. 
+Applying `delete 3 m/transaction` deletes the 3rd transaction of the client.
```

Comment:
Not sure if it is better if we give an example for remark deletion as well. To safeguard ourselves from `insufficient examples` bug report

Reply:
agree. Will add in another variant with `find` followed by `view` and `delete m/remark`